homeassistant/components/tessie/sensor.py

```diff
@@ -0,0 +1,160 @@
+"""Sensor platform for Tessie integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    PERCENTAGE,
+    UnitOfEnergy,
+    UnitOfLength,
+    UnitOfPower,
+    UnitOfPressure,
+    UnitOfSpeed,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN, TessieApi
+from .entity import TessieEntity
+
+PARALLEL_UPDATES = 0
+
+
+DESCRIPTIONS: dict[str, tuple[SensorEntityDescription, ...]] = {
+    TessieApi.CHARGE_STATE: (
+        SensorEntityDescription(
+            name="Battery Level",
```

Comment:
for all of these, name is not needed when you specify translation_key

Reply:
Thanks, I've done that now.